projects/1.18/assets/shadow-x-blocks-plus/blocksplus/lang/zh_cn.json

```
@@ -0,0 +1,190 @@
+{
+    "itemGroup.creativetab": "Blocks +",
+    
+    "container.crafting_table": "合成",
+    
+    "block.blocksplus.bamboo_log": "竹木原木",
+    "block.blocksplus.stripped_bamboo_log": "去皮竹木原木",
+    "block.blocksplus.bamboo_wood": "竹木",
+    "block.blocksplus.stripped_bamboo_wood": "去皮竹木",
+    "block.blocksplus.bamboo_planks": "竹木木板",
+    "block.blocksplus.bamboo_stairs": "竹木楼梯",
+    "block.blocksplus.bamboo_slab": "竹木台阶",
+    "block.blocksplus.bamboo_fence": "竹木栅栏",
+    "block.blocksplus.bamboo_fence_gate": "竹木栅栏门",
+    "block.blocksplus.bamboo_button": "竹木按钮",
+    "block.blocksplus.bamboo_pressure_plate": "竹木压力板",
+    "block.blocksplus.bamboo_door": "竹木门",
+    "block.blocksplus.bamboo_trapdoor": "竹木活板门",
+    "block.blocksplus.bamboo_crafting_table": "竹木工作台",
+    "block.blocksplus.bamboo_bookshelf": "竹木书架",
+    "block.blocksplus.bamboo_ladder": "竹梯",

```

```suggestion
    "block.blocksplus.bamboo_ladder": "竹梯子",

```
这里命名成`竹梯`确实从口语表达来说更顺口（毕竟如果遵循模组内命名的统一性的话本来应该是`竹木梯子`）。但我认为至少可以折衷一下：`竹梯子`？

---

那还是竹木梯子吧